site/src/pages/TemplatePage/TemplatePage.test.tsx

```diff
@@ -12,6 +13,10 @@ import {
 } from "../../testHelpers/renderHelpers"
 import { TemplatePage } from "./TemplatePage"
 
+Object.defineProperty(window, "ResizeObserver", {
+  value: ResizeObserver,
+})
+
```

Comment:
Similar comment as above, would be best to keep this in a setup and teardown function